GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/StillThereViewModel.cs

```diff
@@ -9,8 +9,8 @@ namespace GalaxyZooTouchTable.ViewModels
 {
     public class StillThereViewModel : ViewModelBase
     {
-        public DispatcherTimer SecondTimer { get; set; }
-        public DispatcherTimer ThirtySecondTimer { get; set; }
+        public DispatcherTimer SecondTimer { get; set; } = new DispatcherTimer();
+        public DispatcherTimer ThirtySecondTimer { get; set; } = new DispatcherTimer();
```

Comment:
Not sure if it's worth changing, but it may make more sense to have a counter that counts down from 30 on the `SecondTimer` instead of using the `ThirtySecondTimer`. May make the code more simple if you only had one timer in the code

Reply:
Good call. This will clean up the code quite a bit.